safe-global/web-core
# fix: Add licenses page

## What it solves

Resolves #1737 

## How this PR fixes it

- Adds a new `/licenses` page
- Updates all licenses for web

## ToDos

- [ ] Change `@gnosis.pm/safe-modules-deployments` to  `@safe-global/safe-modules-deployments` since its been deprecated

## How to test it

1. Open the Safe
2. Navigate to `/licenses`
3. Observe a page containing all licenses for Android, iOS and Web

## Screenshots

![Screenshot 2023-03-29 at 15 17 21](https://user-images.githubusercontent.com/5880855/228550339-f4658978-c395-4ff1-8c44-b7bfc9cb4d1a.png)


## Checklist
